tests/console/hwloc_testsuite.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's hwloc tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Run testsuite included in hwloc sources
+# Maintainer: Thomas Blume <tblume@suse.com>
+
+use base "consoletest";
+use warnings;
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    # install hwloc testsuite
+    become_root();
```

Comment:
you should not use become_root though - because that again relies on you being in a user console. The proper fix is to select_console 'root-console' in those tests that need it. In *all of them*